kodaksmarthome/constants.py

```diff
@@ -49,8 +63,8 @@ def __init__(self, region):
     "Accept-Language": "en-GB,en-US;q=0.9,en;q=0.8",
     "Cache-Control": "no-cache",
     "Connection": "keep-alive",
-    "Origin": "https://app-eu.kodaksmarthome.com",
-    "Referer": "https://app-eu.kodaksmarthome.com/web/",
+    "Origin": "https://app-hk.kodaksmarthome.com",
```

Comment:
I see I implemented it wrong, and it adds some limitations to multiple regions.
It needs to be fixed in another way, and actually, the ``HTTP_HEADERS_BASIC``needs to be moved from the ``constants.py`` to the ``KodakSmartHome.api.KodakSmartHome.__init__`` becoming  ``self.headers`` to be used with the reference of region chosen by the API user.

I am open to another suggestion as well.

Reply:
This might work. I'm happy to test out anything that you change. Need to know how to get this to work on my HA install though. Pi4 on HAOS.

Reply:
Please, revert this part.
You will be able to use it when we release the new version or you can do it by changing your system.